services/explorer/src/api/types.ts

```diff
@@ -37,6 +37,13 @@ export type FetchWithPagination = {
   limit: number;
 };
 
+export type FetchWithPaginationResult = {
+  start: number;
+  limit: number;
+  total: number;
+  hasNext: boolean;
+};
+

```

Comment:
We added this 'cause the API returns limit instead of end? Maybe we can change the api

Reply:
Yeah, this is redundant, I can just reuse and extend the one from above